server/internal/worker.go

```diff
@@ -183,9 +186,9 @@ func (w *Worker) start(req *StartReq) (err error) {
 }
 
 func (w *Worker) stop(requestId string, channelName string) (err error) {
-	slog.Info("Worker stop start", "channelName", channelName, "requestId", requestId, logTag)
+	slog.Info("Worker stop start", "channelName", channelName, "requestId", requestId, "pid", w.Pid, logTag)
 
-	err = syscall.Kill(w.Pid, syscall.SIGTERM)
+	err = syscall.Kill(-w.Pid, syscall.SIGTERM)
```

Comment:
why negative?

Reply:
it means to kill all processes under the process group.
as we are starting the command via "sh -c", it's in fact two processes here, which leaves chances that one is killed while the other is left orphan. so we need to group these two processes and kill them as a group.